src/fa-screenshot.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+# Script to take screenshots in the correct format
+
+# how to run : fa-screenshot fileName tp
+# filename (optional)- name of the file to save screenshot to. extension png is automatically added.
+# tp (optional)- use tinypng instead of pngquant for compression
+# TODO: 1. add parameter parsing 2. accept switch for destination folder 3. switch for quieter output
+
+# Author - Sanjay
+
+
+function printFileAttribs() {
+  attributes=`/usr/local/bin/identify ${absFile}`
+  echo "Current dimensions of the screenshot are : "`echo ${attributes} | cut -d' ' -f3`
+  echo "Current size of the screenshot is : "`echo ${attributes} | cut -d' ' -f7`
+}
+
+function compressUsingTPNG() {
+  API_KEY=1QPDBdzWpqs2pRKK9VyJyyzhwVQJ2s5h;
```

Comment:
This repo is public, so you'll want to to rotate this key, and we should probably pick this up from an environment variable or something like that.